packages/sdk/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## Unreleased
+
+### Added
+
+- Introduce function `versionedModuleSourceToBuffer` for serializing a versioned module to a buffer, which can be stored in a file.
+
```

Comment:
Should probably move this to the section including changes for 7.0.0 (which has only been released as a release candidate so far).

You can change the version to "Unreleased" as well, as I think it makes more sense to keep it like that until the official release.